Readme.md

```diff
@@ -0,0 +1,494 @@
+helpful.
+
+Also, we need an [Atlas account][3], which is free forever. Create an account if you haven't got one, that provides MongoDB as a cloud database and
+much more.
+
+In general, MongoDB is an open-source, cross-platform, and distributed document database that allows building apps with flexible schema. In
+case, you are not familiar with or would like a quick recap I would recommend exploring the MongoDB Jumpstart series to get familiar with MongoDB
+and its various services in under 10 minutes or if you prefer to read then you can follow
+this [guide](https://www.mongodb.com/docs/atlas/getting-started/).
+
+And last, to aid our development activities, we would be using [Jetbrains IntelliJ IDEA (Community Edition)](https://www.jetbrains.
+com/idea/download/).
+
+---------------------
+
+## Kotlin driver vs Realm Kotlin SDK ?
+
+Before we start I would like to touch base on [Realm Kotlin SDK](https://www.mongodb.com/docs/realm/sdk/kotlin/), which is another framework by
+MongoDB in Kotlin that is used to create client applications using the MongoDB ecosystem and shouldn't be confused with this. [MongoDB Kotlin driver]
+[4] is used to create backends applications like RESTful applications which can be consumed by client apps.
+
+To make learning more meaningful and practical we would be building a CRUD application, feel free to check out our
+[Github repo][5] if you would like to follow along together. So Without further ado let's
+get started.
+
+---------------------
+
+## Create a project
+
+To create the project we can use the Project wizard which can be found under `File` menu options and then select `New` followed by `Project`. This
+would open `New Project` screen, as shown below, then update the project and language to Kotlin.
+
+![Project wizard](https://images.contentstack.io/v3/assets/blt39790b633ee0d5a7/bltbce8e7adda583e3e/648793838b23a5b3d5052b69/Screenshot_2023-06-12_at_09.34.09.png)
+
+After initial gradle sync our project is ready to run so let's give it a try using the run icon in the menu bar or simply press CTRL + R on mac.
+Currently, our project wouldn't do much apart from printing `Hello World!` and arguments supplied but `BUILD SUCCESSFUL` message in the run
+console is what we're looking which tells us that our project setup is complete.
+
+![build success](https://images.contentstack.io/v3/assets/blt39790b633ee0d5a7/blt97a67a3d4a402196/64879383d40ad08ec16808a9/Screenshot_2023-06-12_at_13.42.38.png)
+
+Now next step is to add the Kotlin driver to our project which would allow us to interact with [MongoDB Atlas][1].
+
+---------------------
+
+## Adding MongoDB Kotlin driver
+
+Adding the driver to the project is simple and straightforward, just update the `dependencies` block in the build file i.e. `build.gradle` with
+these.
+
+```groovy
+dependencies {
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.6.4'
+    implementation 'org.mongodb:mongodb-driver-kotlin-coroutine:4.10.0-SNAPSHOT'
+}
```

Comment:
Might be worth explaining these dependencies so the reader isn't blindly adding things to their project.

Reply:
It's a standard practice for Kotlin/Java developer, also added a code comment to make it super clear.